api/net/Areas/Editor/Controllers/ContentController.cs

```diff
@@ -258,10 +258,11 @@ public async Task<IActionResult> UpdateContentAsync(ContentListModel model)
         {
             if (model.Action == ContentListAction.Publish)
             {
-                if (content.Status != ContentStatus.Published)
+                var latestContent = _contentService.FindById(content.Id);
```

Comment:
I don't understand the need for this.  Line 250 would have received the latest content.

Reply:
Updated, however, I do have a question regarding to the current logic, maybe we can have a quick chat about this? Thanks